INSTALL.md

```diff
@@ -1,27 +1,26 @@
-## Installation (with docker-compose)
+## Installation (with docker compose)
 
-To simply try out mampf you can use `docker-compose` ([needs docker](https://docs.docker.com/engine/install/ubuntu/) && `apt install docker-compose`). Simply clone the mampf repository and run docker-compose by executing
+To simply try out mampf you can use `docker compose` (needs [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/)). Simply clone the MaMpf repository and run docker compose by executing
 ```
-$ git clone -b main --recursive https://github.com/MaMpf-HD/mampf.git
-
+$ git clone -b main --recursive git@github.com:MaMpf-HD/mampf.git
 $ cd mampf/docker/development/
-# docker-compose up
+# docker compose up
 ```
 
-NOTE: Please make sure to clone recursively as the pdf compression feature is in an extra repository.
-If you have an already checked out version simply run:
+NOTE: Please make sure to clone recursively as the pdf compression feature is in a separate repository.
 
-```sh
+If you have an already checked out version simply run:
+```
 git submodule init
 git submodule update
 ```
 
 You now have the following things ready:
-* The MaMpf server on <a href="http://localhost:3000/" target="_blank">localhost:3000</a>
-* The mailcatcher service on <a href="http://localhost:1080/" target="_blank">localhost:1080</a>
-* The webinterface for ApacheSolr on <a href="http://localhost:8983/" target="_blank">localhost:8983</a>
+* The MaMpf server on [localhost:3000](http://localhost:3000/)
+* The mailcatcher service on [localhost:1080](http://localhost:1080/)
+* The webinterface for ApacheSolr on [localhost:8983](http://localhost:8983/)
 * A test mailserver instance on Ports 1025, 10143, 10993
-* A webpacker dev server on localhost:3035
+* A webpacker dev server on [localhost:3035](http://localhost:3035/)
```

Comment:
Please undo the changes to this section, automatically having new tabs opened via `target="_blank"` is pretty handy 😊

Reply:
However, GitHub markdown does not reflect this, right?